modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -3533,10 +3539,10 @@
                         "servicePeriods": [
                           {
                             "serviceBranch": "Public Health Service",
-                            "activeDutyBeginDate": "11-14-2015",
-                            "activeDutyEndDate": "11-30-2018",
+                            "activeDutyBeginDate": "11-14-2023",
+                            "activeDutyEndDate": "11-30-2025",
```

Comment:
interesting that these can be in the future. is that what's intended as well as what's happening?